### ISSUE TITLE
The main `settings` is not applied

### ISSUE THREAD

**Issue by eunjae-lee** [algolia/gatsby-plugin-algolia#64]
~Unlike query.indexName, query.settings doesn't override the main settings. Is it intended?~

**Comment by eunjae-lee** [comment]
The title and the description were wrong 😅
So what happened is, 

```js
{
  resolve: 'gatsby-plugin-algolia',
  options: {
    appId,
    apiKey,
    indexName,
    queries,
    settings,
  },
},
```

That `settings` doesn't work.
For me to update the settings, I had to specify it in `query.settings`.
I don't think that main `settings` is passed to `getSettingsToApply`.

**Comment by Haroenv** [comment]
It's too bad I didn't find a way to add tests to this repo yet, otherwise this would be easy to prove with a new test

**Comment by Haroenv** [comment]
fixed in 0.11.0
